test/test_callbacks.jl

```diff
@@ -44,13 +45,30 @@ simulation = Simulation(
     model,
     SSPRK33(),
     dt = 0.01,
-    tspan = (0.0, 0.02),
+    tspan = (0.0, 0.03),
     callbacks = cb_set,
 )
 run!(simulation)
 
-@test isfile(joinpath(cb_1.filedir, cb_1.filename * ".jld2")) == true
-@test isfile(joinpath(cb_2.filedir, cb_2.filename * ".jld2")) == true
+@info "Testing Simulation Restart"
+simulation = Simulation(
+    model,
+    SSPRK33(),
+    dt = 0.01,
+    tspan = (0.0, 0.03),
+    callbacks = cb_set,
+    restart = Restart(
+        restartfile = joinpath(cb_1.filedir, cb_1.filename * "_0.02.jld2"),
+        end_time = 0.05,
```

Comment:
Just to make sure, if using restart, t_span[2] doesn't matter anymore and the end_time determines the end of the simulation, right?

Reply:
Yes, if `Restart` is defined, then `Restart.end_time` determines the final solution time.